test/e2e/mcq.cy.js

```diff
@@ -0,0 +1,26 @@
+describe('Multiple Choice Question', function () {
+  beforeEach(function () {
+    cy.getData()
+    cy.visit('/');
+  });
+
+  it('should display the mcq component', function () {
+    const mcqComponents = this.data.components.filter((component) => component._component === 'mcq')
+    mcqComponents.forEach((mcqComponent) => {
```

Comment:
We can remove the additional brackets from the arrow functions. 

```suggestion
    const mcqComponents = this.data.components.filter(component => component._component === 'mcq')
    mcqComponents.forEach(mcqComponent => {
```